source/_components/climate.hive.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: "Hive Thermostat"
+description: "Instructions on how to integrate Hive thermostat(s) with Home Assistant."
+date: 2017-09-24 21:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hive.png
+ha_category: Climate
+ha_release: 0.62
```

Comment:
You might wanna update the version number (in all other files as well)

Reply:
@frenck 
Thanks for reviewing.
I did not know which version number the code PR for the Hive component / platform would actually be released in, so I put a prediction.

What version would you recommended to put here, I don't anticipate the code being accepted any time soon?

Reply:
Aah ok, that makes sense I guess. We usually put in the next version (which will be `0.59`)

Reply:
I can update to 0.59 and hope for the best :)